tests/api/test_service.py

```diff
@@ -0,0 +1,132 @@
+import allure
+import pytest
+
+from data.models import AllEntitiesModel, EntityModel
+from lib.service import create_entity, delete_entity, get_all_entities, get_entity, should_be_deleted, update_entity
+
+
+@allure.feature("Service")
+@allure.story("API")
+@allure.title("Создание сущности")
+@allure.description(
+    """
+        Шаги:
+            1. Создать сущность
+            2. Проверить создание сущности
+        
+        Постусловие:  
+            - Удалить тестовые данные
+        
+        Ожидаемый результат:  
+            - Сущность создана
+    """
+)
+def test_create_entity(entity):
+    with allure.step("Создание сущности"):
+        entity_id = create_entity(entity)
+
+    with allure.step("Проверка создания сущности"):
+        assert entity_id == EntityModel(**get_entity(entity_id)).id
+
+    with allure.step("Удаление тестовых данных"):
+        delete_entity(entity_id)
```

Comment:
Возможна такая ситуация, что на каком-то из шагов после создания сущности у нас возникнет ошибка и при этом тест завершит выполнение, а шаги, которые находятся внутри тестовой функции, не выполнятся. Поэтому предлагаю использовать блок `try except`.
```suggestion
    try:
        with allure.step("Создание сущности"):
            entity_id = create_entity(entity)
        
        with allure.step("Проверка создания сущности"):
            assert entity_id == EntityModel(**get_entity(entity_id)).id
    except:
        with allure.step("Удаление тестовых данных"):
            delete_entity(entity_id)
```